schema-lending.graphql

```diff
@@ -82,6 +91,17 @@ enum LendingType {
   POOLED
 }
 
+enum PermissionType {
+  " Protocols that only allow KYC'd users "
+  PERMISSIONED
```

Comment:
I think this might need to be more nuanced. For example, there are protocols that allow *any* KYC-ed users, but some that allow only whitelisted users (e.g. institutions).

Also, I think we should describe permission for lender and borrowers separately. For example, a protocol can allow anyone to lend, but only KYCed users to borrow.